### PR TITLE
Informa bg changes

### PR DESCRIPTION
**Pull request by crandall-endeavor**
https://southcomm.atlassian.net/browse/DEV-348

Informa buyers guide changes to move and prefix company attribution, and add articles and products to directory pages."

Screen Shot 2020-11-12 at 11 22 05 AM

Screen Shot 2020-11-12 at 9 45 50 AM

### CODE REVIEW

**Review by CaitlynKnudsvig — COMMENTED** [review]
Is TDworld the only group that doesn't want the company name to display? If not then we'll have to remove the element in @base-cms/base-cms

**Comment by crandall-endeavor** [comment]
The source of the request came from TDWorld, but they may want it everywhere. The problem is that P1 isn't going to approve removing it in their codebase.

**Comment by crandall-endeavor** [comment]
To clarify, they originally wanted to move the company attribution and relabel it, but that's a bigger base-cms change that will definitely not get approved.

**Comment by zarathustra323** [comment]
@crandall-endeavor and @CaitlynKnudsvig removing the company name from the content page attribution does not require a change to the `base-cms` repo, nor do you need to hide it via CSS. The `<default-theme-content-attribution>` component supports an `elements` input that allows you to customize the items that will be displayed (and in what order). See https://github.com/base-cms/base-cms/blob/c2155aeae3fe97e709607e94468743667bb348f8/packages/marko-web-theme-default/components/content/attribution/index.marko#L11

In this case, you'd pass a new `elements` input array excluding `company` to the attribution component found in the `content-page-card-node` found here: https://github.com/base-cms-websites/informa-business-intelligence/blob/5ea171fa6846468802c7b60da804107444a32302/packages/lazarus-shared/components/nodes/content-page-card.marko#L86

In other words:
`<default-theme-content-attribution obj=content elements=["byline", "authors", "contributors", "photographers", "source", "sponsors"] />`

**Comment by crandall-endeavor** [comment]
Ah, that's good to know! Thanks! I'm waiting for final confirmation as Alex would like to run it by some people first.

**Comment by crandall-endeavor** [comment]
Ok @zarathustra323 , your feedback actually allowed me to do exactly what they asked for, which was to add Related To: to the company and move it below the post date.

`**<default-theme-content-attribution obj=content elements=["byline", "authors", "contributors", "photographers", "source", "sponsors"] />**
        <default-theme-page-dates|{ blockName }|>
          <if(type === "event")>
            <marko-web-content-start-date block-name=blockName obj=content />
            <marko-web-content-end-date block-name=blockName obj=content />
          </if>
          <if(type === "webinar")>
            <marko-web-content-start-date block-name=blockName obj=content />
          </if>
          <if(displayPublishedDate)>
            <marko-web-content-published block-name=blockName obj=content />
          </if>
        </default-theme-page-dates>
        **<div class="company-attribution"><div class="company-attribution__prefix">Related To: </div><default-theme-content-attribution obj=content elements=["company"] /></div>**`

**Review by CaitlynKnudsvig — APPROVED** [review]
